requirements.txt

```diff
@@ -15,6 +15,7 @@ betamax==0.8.1
 coverage==6.3.2
 pytest-cov==3.0.0
 coveralls==3.3.1
+Pillow==5.2.0
```

Comment:
This is needed for CI right? Not as a requirement for pykechain. If not we need also to update setup.py

Reply:
That's right, will move it to the right requirements file